packages/binding-modbus/README.md

```diff
@@ -3,62 +3,98 @@
 ## Overview
 
 W3C Web of Things (WoT) Protocol Binding for [Modbus](https://en.wikipedia.org/wiki/Modbus) TCP [RFC](https://tools.ietf.org/html/draft-dube-modbus-applproto-00).
-This package uses [modbus-serial](https://www.npmjs.com/package/modbus-serial) as a low level client for Modbus TCP.
+This package uses [modbus-serial](https://www.npmjs.com/package/modbus-serial) as a low-level client for Modbus TCP.
 W3C WoT Binding Template for Modbus can be found [here](https://w3c.github.io/wot-binding-templates/bindings/protocols/modbus/index.html).
 
 Current Maintainer(s): [@relu91](https://github.com/relu91) [@fillobotto](https://github.com/fillobotto)
 
-## Protocol specifier
+## Client Example
+
+You can use a code like the following to use the binding. This specific code is interacting with one of the Eclipse Thingweb Test Things at <https://github.com/eclipse-thingweb/test-things/tree/main/things/elevator>.
```

Comment:
Note: Pointing to other repos/folders makes it even more likely that it gets out of sync but I do not have a better proposal.

Reply:
My long-term vision is to have those test things very stable with respect to the binding specs and even to use them for some end-to-end tests. For me, it is similar to pointing to to hosted things we have such as we do at https://github.com/eclipse-thingweb/node-wot/tree/master/packages/binding-http#client-example